enterprise/api_client/ecommerce.py

```diff
@@ -43,25 +40,3 @@ def __init__(self, user):
 
         self.user = user
         self.client = ecommerce_api_client(user)
-
-    def get_course_final_price(self, mode, currency='$', enterprise_catalog_uuid=None):
```

Comment:
although the comment implies this is specific to entitlements, i think it's lying. it seems like this is generally trying to get discounted prices from ecommerce, and this is still relevant to the enterprise enrollment workflow. i would keep this in.